### PR TITLE
Suggestion

### PR DESCRIPTION
**Pull request by matot830**
Hi friend!

I have made great changes. Accept them now.

All the best,
Anonymus

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
 theme: jekyll-theme-minimal
 title: Test Testson
-description: My portfolio website
-
+description: My bestest portfolio website
```
Comment (Grille1992) [inline _config.yml:3]:
```suggestion
description: My own portfolio website
```

**Review by Grille1992 — CHANGES_REQUESTED** [review]
Tack men nej tack

File: _config.yml (hunk 1 above)
Comment (matot830) [reply _config.yml:3]: Strong argument. I accept!

**Review by Grille1992 — APPROVED** [review]
Perfekt

**Pull request merged**